Main dev

bx44 opened the pull request:
The branch includes all the files of the project.

bx44 committed:
Added basic files and partial structure of index.html

bx44 committed:
Added image to header and styled content

bx44 committed:
Styled row of thumbnails

bx44 committed:
Styled and corrected head nav bar

bx44 committed:
Added footer and styled using flexbox

bx44 committed:
Validated html and fixed css

bx44 committed:
Updated README.md file

shshamim63 commented:
## Status :recycle: [!! Required Changes !!] :recycle: 
___
Hi @bx44  :wave:,
Well done. Code is neat and clean. Constructive readme, where it provides the basic idea and the purpose of this project :clap:. But there are some issues that are actually blocking me to approve this project. And i am sure that you will fix them easily :sunglasses:.
___
#### Please follow the instructions below :point_down: to complete this project.
___
### :spades: [Mandatory Changes] :spades: 
___
1. The component inside the nav are actually Buttons and a input field(Based on the actual web-page). But on your one those are images.
https://github.com/bx44/apple-landing-page/blob/394ddeeb0ea0f3bb3103d113df6815f9dcf1857a/index.html#L11-L19
2. The footer should contain more space on top and bottom.
![image](https://user-images.githubusercontent.com/38625956/70177870-5fef1200-1705-11ea-806a-2b3851fd36c1.png)
___
### :clubs: [Optional Changes] :clubs: 
___
1. The actual page consist the 100%. But you have used some padding. Well I think that the actual one is really eye catching than the formation you have.
https://github.com/bx44/apple-landing-page/blob/394ddeeb0ea0f3bb3103d113df6815f9dcf1857a/style.css#L1-L6
___
#### Please fix the above mentioned issue and send another review request. :pray: 
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

bx44 committed:
Changes upon review

-  Removed margin from body,  and added max-width to avoid over streching.
- Added margin to footer.
- Added style to nav buttons and an input field.

bx44 committed:
Added padding to footer

shshamim63 commented:
## Status :taco: :tada:  [!! Approved !!] :tada: :taco:  
___
Hi @bx44  :wave:,
Awesome job :clap: . You have covered all the changes mentioned previously. Well done.
___
####  This :+1: is from me. You can merge it. And don't forget to fill the project completion form.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)
